Sources/TuistKit/Services/ScaffoldService.swift

```diff
@@ -136,6 +144,17 @@ class ScaffoldService {
         }
     }
 
+    /// Locates all template directories, local, system, and plugin.
+    /// - Parameter path: The path where the command is being executed.
+    /// - Returns: A list of template directories
+    private func locateTemplateDirectories(at path: AbsolutePath) throws -> [AbsolutePath] {
+        let config = try configLoader.loadConfig(path: path)
```

Comment:
Nitpick: What about moving the loading of the manifests to the `run` method and then pass it down? If we do it in private methods, we risk doing the same in another method and therefore doing unnecessary loads.